templates/openstack/support.html

```diff
@@ -0,0 +1,530 @@
+{% extends "openstack/_base_openstack.html" %}
+
+{% block title %}OpenStack support on Ubuntu{% endblock %}
+
+{% block meta_description %}Canonical is a global leader in OpenStack support, offering enterprise full-stack support subscription at the best server/annum cost.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1mHaZQ_VZnISTUEpYKiVK-Ejr5ne8lVevpxEc0LpXYKI/edit#heading=h.wtb1j7wwfobn{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="u-fixed-width">
+    <h1>OpenStack support</h1>
+    
+    <p class="p-heading--4">Use OpenStack with confidence with Ubuntu Advantage for Infrastructure</p>
+    
+    <p>One package, multiple benefits. The most comprehensive Linux enterprise subscription, covering all aspects of open infrastructure.</p>
+    
+    <p>Subscribe to Ubuntu Advantage for Infrastructure and benefit from extended security, certified compliance and 24x7 support.</p>
+
+    <a href="/openstack/support#get-in-touch" class="p-button--positive js-invoke-modal">Get OpenStack support</a>
+
+    <a href="/engage/redhat-openstack-comparison-whitepaper">Read a Red Hat OpenStack comparison whitepaper&nbsp;&rsaquo;</a>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
```

Comment:
Should use the p-logo-section vanilla pattern here https://vanillaframework.io/docs/patterns/logo-section

Reply:
Good point!